src/Raven.Server/NotificationCenter/Notifications/Details/LicenseLimitWarning.cs

```diff
@@ -38,9 +38,15 @@ public static void AddLicenseLimitNotification(NotificationCenter notificationCe
                 licenseLimit.Message,
                 AlertType.LicenseManager_LicenseLimit,
                 NotificationSeverity.Warning,
+                key: licenseLimit.Type.ToString(),
                 details: new LicenseLimitWarning(licenseLimit));
 
-            notificationCenter.Add(alert);
+            notificationCenter.Add(alert, updateExisting: false);
```

Comment:
I've set `updateExisting` to `false` to avoid opening a write tx each time. The only 'problem' is that the 'created' date will not be changed. Not sure if this is something that we should worry about? @grisha-kotler

Reply:
It should be very rare (when you start your server)